google/cloud/storage/_http.py

```diff
@@ -31,10 +37,11 @@ class Connection(_http.JSONConnection):
     :param client_info: (Optional) instance used to generate user agent.
 
     :type api_endpoint: str
-    :param api_endpoint: (Optional) api endpoint to use.
+    :param api_endpoint: (Optional) api endpoint to use. This may be a universe
+        domain if one was specified.
```

Comment:
Does this mean that a universe domain can also be set via `Connection.api_endpoint`, in addition to `client_options.universe_domain`?

Reply:
This docstring is confusing. I'll rewrite it. Basically, users should never interact with this directly, but universe_domain being set may affect how our system sets this parameter.